mysensors/mysensors.py

```diff
@@ -368,6 +376,12 @@ def set_child_value(self, child_id, value_type, value):
                             sub_type=value_type, payload=value)
         return None
 
+    def get_child_value(self, child_id, value_type):
```

Comment:
I don't think you need this method. You have access to the values in children in the sensors attribute in the Gateway class.